NetUtilities.Tests/System/Collections/Generics/ReadOnlyListTests.cs

```diff
@@ -21,7 +21,7 @@ public void List_Doesnt_Mutate_If_Reference_Does()
         public void Indexing_Works()
         {
             var list = new int[] { 1, 2, 3, 4, 5, 6, 7 }.ToReadOnlyList();
-            var test = list[0..3];
+            var test = list[0..2];
```

Comment:
This has to be 3 because Range is meant to be upper bound exclusive. I actually noticed that I made it wrong on ReadOnlyList#Slice, can you update it?

change this: `for (var count = 0; count <= length; count++)`
to this: `for (var count = 0; count < length; count++)`

Reply:
Ok, it makes sense. I was misunderstanding how slices work in .net (offset -> length) and thinking in an ADA-like style (left_bound -> right bound).